cmd/fleetctl/query.go

```diff
@@ -116,7 +130,19 @@ func queryCommand() cli.Command {
 							percentOnline = 100 * float64(responded) / float64(online)
 						}
 					}
-					s.Suffix = fmt.Sprintf("  %.f%% responded (%.f%% online) | %d/%d targeted hosts (%d/%d online)", percentTotal, percentOnline, responded, total, responded, online)
+
+					if percentTotal >= 100 && flExit {
```

Comment:
I'd prefer to compare the actual values rather than the percentages.

Reply:
Updated